content/90.back-matter.md

```diff
@@ -198,7 +198,8 @@ For this reason we use the measured precision instead of the predicted probabili
 | hematologic cancer           | TNF         | the mean [tnf-alpha]{.gene_color} plasma concentration in the patients with [cll]{.disease_color} was significantly higher than in the healthy control population ( 16.4 versus 8.7 pg/ml ; p < .0001 ) .                                                                                                                                          | 0.999              | 0.81              | 
 | lung cancer                  | MUC16       | the mean concentration of [ca 125]{.gene_color} was higher in patients with [lung cancer]{.disease_color} ( 37 + / - 81 u/ml ) than in those with nonmalignant disease ( 4.2 + / - 5.7 u/ml ) ( p less than 0.01 ) .                                                                                                                               | 0.999              | 0.806             | 
 | prostate cancer              | AR          | the [androgen receptor]{.gene_color} was expressed in all primary and metastatic [prostate cancer]{.disease_color} tissues and no mutations were identified .                                                                                                                                                                                      | 0.999              | 0.801             | 
-| breast cancer                | ERBB2       | the results of multiple linear regression analysis , with her2 as the dependent variable , showed that family history of [breast cancer]{.disease_color} was significantly associated with elevated [her2]{.gene_color} levels in the tumors ( p = 0.0038 ) , after controlling for the effects of age , tumor estrogen receptor , and dna index . | 0.999              | 0.8               |  
+| breast cancer                | ERBB2       | the results of multiple linear regression analysis , with her2 as the dependent variable , showed that family history of [breast cancer]{.disease_color} was significantly associated with elevated [her2]{.gene_color} levels in the tumors ( p = 0.0038 ) , after controlling for the effects of age , tumor estrogen receptor , and dna index . | 0.999              | 0.8               | 
```

Comment:
Did these not change at all? It seems like your models have changed a good bit, so I'm surprised this back matter is unaffected. Maybe it gets changed in a different PR I haven't made it to yet.